assets/js/components/Beneficiaries.vue

```diff
@@ -69,6 +69,66 @@
   }
 }
 
+.d3-tip.benef {
```

Comment:
Please extract the CSS from Beneficiaries.vue in its own resource, we will probably reuse it for the other components